frontend/e-buddy/src/app/(demo)/bots/setup/page.tsx

```
@@ -1,153 +1,251 @@
-                <button 
-                  onClick={prevStep} 
-                  className="bg-gray-300 text-gray-700 px-6 py-3 rounded-lg hover:bg-gray-400 transition-colors"
+                <p className="text-xs text-gray-500 mt-2">
+                  We'll extract data from this website to train your bot
+                </p>
+              </div>
+              <div className="mt-10 flex justify-between">
+                <button
+                  onClick={prevStep}
+                  className="btn-secondary flex items-center gap-2"
                 >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)